src/chosungIncludes.ts

```diff
@@ -1,23 +1,29 @@
-import { HANGUL_CHARACTERS_BY_FIRST_INDEX } from './constants';
 import { disassembleHangulToGroups } from './disassemble';
-import { canBeChosung, getFirstConsonants, hasValueInReadOnlyStringList } from './utils';
+import { canBeChosung, getFirstConsonants } from './utils';
 
 export function chosungIncludes(x: string, y: string) {
-  if (!isOnlyInitialConsonant(y)) {
+  const trimY = y.replace(/\s/g, '');
```

Comment:
마이너 합니다! 잘린 그 후 를 나타내는 의미로 과거형을 쓰는것도 좋아보여요!

```suggestion
  const trimmedY = y.replace(/\s/g, '');
```

Reply:
피드백 감사합니다! 훨씬 적절한 것 같습니다 😁

Reply:
아래 이 변수가 사용되는 두개도 고쳐주면 좋을 것 같습니당!